mava/utils/logger_tools.py

```diff
@@ -172,6 +173,7 @@ def __init__(
         environment_name: str,
         seed: int,
     ):
+        self.start_time = time.time()
```

Comment:
Will this not always be overwritten? We init the logger, and `self.start_time = time.time()`, but then when `evaluation_step == 0`, we set it again.

Perhaps it's better here to write something like,
```suggestion
        self.start_time = -1
```

That way, it will always be overwritten later, & if there's a bug, we can notice it easily? :)